benchmarks/src/main/scala/zio/redis/RedisClients.scala

```diff
@@ -0,0 +1,66 @@
+package zio.redis
+
+import cats.effect.{ Blocker, Resource, IO => CatsIO }
+import dev.profunktor.redis4cats.RedisCommands
+import dev.profunktor.redis4cats.data.RedisCodec
+import fs2.io.tcp.SocketGroup
+import io.chrisdavenport.rediculous.{ Redis, RedisConnection }
+import io.lettuce.core.ClientOptions
+import laserdisc.fs2.RedisClient
+import zio.redis.BenchmarkRuntime.{ RedisHost, RedisPort }
+
+object RedisClients {
+  import BenchmarkRuntime.cs
+  import BenchmarkRuntime.timer
+
+  type RedisIO[A] = Redis[CatsIO, A]
+
+  type Redis4CatsClient[V] = RedisCommands[CatsIO, String, V]
+  type LaserDiskClient     = RedisClient[CatsIO]
+  type RediculousClient    = RedisConnection[CatsIO]
+
+  trait QueryUnsafeRunner[F] {
+    def unsafeRun(f: F => CatsIO[Unit]): Unit
+  }
+
+  implicit object LaserDiskClientRunner extends QueryUnsafeRunner[LaserDiskClient] {
```

Comment:
Maybe put instances into companion 9f QueryUnsafeRunner. That should remove any imports.